### ISSUE TITLE
Traefik needs .env file to properly name its network

### ISSUE THREAD

**Issue by jtreminio** [jtreminio/dashtainer#1]
Per https://twitter.com/ejunker/status/981541773901213696

Traefik is creating its network based on directory name!

**Comment by ejunker** [comment]
Ok, I figured out how to fix this in the docker-compose.yml. Need to bump version to `3.5` and then you can specify the network name as `traefik_webgateway`

```
version: '3.5'

services:
  proxy:
    image: traefik
    command: --api --docker --docker.domain=docker.localhost --logLevel=DEBUG
    networks:
      - webgateway
    ports:
      - "80:80"
      - "8080:8080"
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - /dev/null:/traefik.toml

networks:
  webgateway:
    name: traefik_webgateway
    driver: bridge

```

**Comment by jtreminio** [comment]
Fixed on prod. You can patch in fix from 1a7b9c42646f2d6b8b7051ff0400d36f34c277df or redownload zip file.

**Comment by Cyberiaaxis** [comment]
awesome
